docs/product/user-feedback/index.mdx

```diff
@@ -86,3 +101,19 @@ If you don't have Sentry's default issue alert turned on, follow these steps to
 ![User Feedback Alert Configuration.](./img/user-feedback-alert-configuration.png)
 
 To get notifications when crash-report feedback comes in, make sure to turn on "Enable Crash Report Notifications" in Settings > Projects > [Project Name] > User Feedback.
+
+## Automated Issue Creation for User Feedback
+
+If you'd like to automatically create a GitHub or Jira issue every time you receive a user feedback, you can do so through the Alerts page in Sentry, similar to how you would create an issue alert for user feedback (illustrated in the section above).
+
+Follow these steps to set up automated GitHub or Jira issue creation for every new feedback:
+
+1. Create a [New Alert Rule](https://sentry.io/alerts/new/issue/) in Sentry.
+2. Scroll to the "Set conditions" section and set the "IF" filter to `The issue's category is equal to… "Feedback"`.
+3. Select `Create a GitHub issue` or `Create a Jira issue` under the "THEN" filter.
```

Comment:
Oh that is, unless we only support GH and Jira?

Reply:
i only tested on sentry, but we offer others as well. i'll update the wording